src/HoJBot.jl

```diff
@@ -22,6 +22,10 @@ import HTTP
 import JSON3
 import StructTypes
 
+include("pluginbase.jl")
+
+import .PluginBase: handle_command as handler, handle_observer as commander
```

Comment:
this won't be necessary anymore once we moved every plugin over to the modular system.